### ISSUE TITLE
ADA Compliant Issues

### ISSUE THREAD

**Issue by SayedTareq** [elementor/elementor#14787]
I got some ADA Compliant Issues which are below

### Incorrect focus order

The focus order on Elementor modal is incorrect as it is moving from right to left. The logical way should be from left to right.
Correct: 
**1. Fullscreen
2. Zoom
3. share 
4. close** 

WCAG - 2.4.3 Focus Order: https://www.w3.org/WAI/WCAG21/Understanding/focus-order

### Insufficient error message provided

Elementor every required form fields need to have a custom error message option. Currently, there have a custom message option globally but every specific required field needs to be a custom error message option so that screen reader can read it so that screen reader users can understand what should they need to do.

WCAG - 3.3.3 Error Suggestion: https://www.w3.org/WAI/WCAG21/Understanding/error-suggestion.html

### Notification not announced - The screen reader doesn't announce the slide changes notification to the user.

**aria-live="polite" should be used on <div class="swiper-wrapper"> and the hidden slide should have "visibility:hidden"
Also, a play/pause button should be there**

**Comment by shilo-ey** [comment]
Thank you for sharing this with us

We will review it and issue fixes for this

Thanks

**Comment by SayedTareq** [comment]
Also please check this issue. I think Elementor use smooth scrolling function. When link a button to a section the focus isn't move on the selected section. Please check this page https://accessdesignstudio.com/speaking-media/ at the top Book Denise button linked to the bottom footer section. When click on that button it goes to that section but the focus not move to that section. The focus needs to be move to the selected section as WCAG 2.1 - Understanding Success Criterion 2.4.3: Focus Order - https://github.com/elementor/elementor/labels/type%2Faccessibility

**Comment by SayedTareq** [comment]
Please add play/pause button on Elementor slider as it need according to this https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html
